packages/pnpm/src/cmd/outdated.ts

```diff
@@ -88,42 +91,72 @@ export default async function (
 
   if (!outdatedPackages.length) return
 
-  let columnNames = [
-    'Package',
-    'Current',
-    'Latest'
-  ]
+  if (opts.table !== false) {
+    let columnNames = [
+      'Package',
+      'Current',
+      'Latest'
+    ]
 
-  if (opts.long) {
-    columnNames.push('Details')
-  }
+    if (opts.long) {
+      columnNames.push('Details')
+    }
 
-  columnNames = columnNames.map((name: string) => chalk.blueBright(name))
-  let columnFns: Array<(outdatedPkg: OutdatedWithVersionDiff) => string> = [
-    renderPackageName,
-    renderCurrent,
-    renderLatest,
-  ]
+    columnNames = columnNames.map((name: string) => chalk.blueBright(name))
+    let columnFns: Array<(outdatedPkg: OutdatedWithVersionDiff) => string> = [
+      renderPackageName,
+      renderCurrent,
+      renderLatest,
+    ]
+
+    if (opts.long) {
+      columnFns.push(renderDetails)
+    }
+
+    return table([
+      columnNames,
+      ...sortedPackages(outdatedPackages).map((outdatedPkg) => columnFns.map((fn) => fn(outdatedPkg))),
+    ], TABLE_OPTIONS)
+  } else {
+    return sortedPackages(outdatedPackages)
+      .map((outdatedPkg) => {
+        let info = stripIndent`
+          ${renderPackageName(outdatedPkg)}
+          ${renderCurrent(outdatedPkg)} => ${renderLatest(outdatedPkg)}`
+
+        if (opts.long) {
+          const details = renderDetails(outdatedPkg)
 
-  if (opts.long) {
-    columnFns.push(renderDetails)
+          if (details) {
+            info += `\n${details}`
+          }
+        }
+
+        return info
+      })
+      .join('\n\n') + '\n'
   }
+}
 
-  return table([
-    columnNames,
-    ...R.sortWith(
-      [
-        sortBySemverChange,
-        (o1, o2) => o1.packageName.localeCompare(o2.packageName),
-      ],
-      outdatedPackages.map(toOutdatedWithVersionDiff)
-    )
-      .map((outdatedPkg) => columnFns.map((fn) => fn(outdatedPkg))),
-  ], TABLE_OPTIONS)
+function sortedPackages (outdatedPackages: Array<OutdatedPackage>) {
```

Comment:
sortPackages would be the right function name.
Maybe even sortOutdatedPackages

Reply:
No, because **sort**Something implies sorting in place. This is returning a sorted copy, which **sorted** implies. Also, it's intended to be used like this:

```js
for (const pkg of sortedPackages(outdatedPackages))
```

which is more semantic than

```js
for (const pkg of sortPackages(outdatedPackages))
```